bin/install-from-cache.js

```diff
@@ -138,6 +138,13 @@ const write = async (name, data) => {
 
 const main = async () => {
   checks: {
+    if (!process.env.npm_package_json && process.env.PWD) {
```

Comment:
Just to note: Yarn 2 does not set the `process.env.PWD` value, that is set by your terminal